pkg/webhook/utils.go

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhook
+
+import (
+	"github.com/pkg/errors"
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission/builder"
+)
+
+const (
+	// NameSuffix is a common suffix for all webhook names.
+	NameSuffix = "extensions.gardener.cloud"
+	// SeedProviderLabel is a label on shoot namespaces in the seed cluster that identifies the Seed provider.
+	SeedProviderLabel = "seed.garden.sapcloud.io/provider"
+	// ShootProviderLabel is a label on shoot namespaces in the seed cluster that identifies the Shoot provider.
+	ShootProviderLabel = "shoot.garden.sapcloud.io/provider"
```

Comment:
```suggestion
	ShootProviderLabel = "shoot.gardener.cloud/provider"
```
Also, this constant should be added to gardener/gardener and used from there (can you add a TODO here?)

Reply:
Done.